### ISSUE TITLE
Validation Accuracy - Finetuning ROBERTA on Glue Tasks

### ISSUE THREAD

**Issue by prishasatwani** [facebookresearch/fairseq#4577]
Hi,

When finetuning the Roberta model on GLUE tasks as mentioned [here](https://github.com/facebookresearch/fairseq/blob/main/examples/roberta/README.glue.md), what does the validation accuracy (that comes up after each epoch) represent? It is different to the accuracy I get when I carry out inference so would the validation accuracy be for the test data as opposed to the dev data?

Thank you.
